opm/parser/eclipse/EclipseState/Grid/EclipseGrid.hpp

```diff
@@ -216,6 +217,8 @@ namespace Opm {
         int m_nactive;
         std::vector<int> m_active_to_global;
         std::vector<int> m_global_to_active;
+        // Numerical aquifer cells, needs to be active
+        std::unordered_set<int> m_num_aquifer_cells;
```

Comment:
This should be `std::size_t` and not `int`

Reply:
The variable should be e.g. `m_aquifer_cells` - it is not a *num_xxx*

Reply:
do you prefer `m_aquifer_cells` or `m_numerical_aquifer_cells`? It is specific for numerical aquifers.